src/video/wayland/SDL_waylandwindow.c

```diff
@@ -886,6 +930,7 @@ static void decoration_frame_configure(struct libdecor_frame *frame,
         fullscreen = (window_state & LIBDECOR_WINDOW_STATE_FULLSCREEN) != 0;
         maximized = (window_state & LIBDECOR_WINDOW_STATE_MAXIMIZED) != 0;
         focused = (window_state & LIBDECOR_WINDOW_STATE_ACTIVE) != 0;
+        suspended = (window_state & (1 << 7)) != 0; /* LIBDECOR_WINDOW_STATE_SUSPENDED */
```

Comment:
Can you do `#ifndef X, #define X` at the top of the file for this?

Reply:
These are defined as enums in the libdecor header, so we would need to check for it via CMake, and since it's an enum, that would get a bit ugly, since `CheckSymbolExists` doesn't work on enums.

Let's just drop it for now, and when libdecor gets the necessary patch merged and a version bump with this functionality, we can just check the version and define something like `LIBDECOR_HAS_STATE_SUSPENDED` in the CMake script if the version being built against is high enough.